ansible/roles_ocp_workloads/ocp4_workload_openshift_ai/tasks/workload.yml

```diff
@@ -33,6 +33,37 @@
     msg: r_ds_cluster
     verbosity: 2
 
+- name: Configure rhods-dashboard deployment replicas
+  when:
+    - ocp4_workload_openshift_ai_deploy_dsc | bool
+    - ocp4_workload_openshift_ai_dashboard == "Managed"
+    - ocp4_workload_openshift_ai_dashboard_replicas is defined
```

Comment:
I believe only this condition would be necessary:

ocp4_workload_openshift_ai_dashboard_replicas is defined

Adding the other ones creates a unnecessary coupling. I think if this last condition is true, the other first two condition will always be true.

Not a big deal, fell free to adjust if you wish.

Reply:
thanks for the feedback, i have removed them as you suggested.